build/Dockerfile

```diff
@@ -3,6 +3,12 @@ MAINTAINER Ansible Service Broker Community
 
 RUN useradd -M -s /usr/sbin/nologin -u 1001 -g 0 ansibleapp
 
+RUN yum -y update \
```

Comment:
I remember adding it first because the other packages didn't install. It was a while ago.